HISTORY.md

```diff
@@ -1,12 +1,4 @@
-## Version 5.6.0
-### Summary
-This is an unsupported Release. Add 2 features as listed below.
-
-### Features
-- Allow ha-sync-batch-size for rabbitmq_policy definition to be integer
-- Ability to set management_hostname in rabbitmqadmin.conf
-
-## Version 5.5.0
+## 5.5.0 (2016-08-29)
```

Comment:
This `$RELEASE ($DATE)` format is needed for consistency *and* so the github-changelog-generator detects version 5.5.0 correctly.